pkg/controller/plan/kubevirt.go

```diff
@@ -2102,16 +2104,20 @@ func (r *KubeVirt) EnsurePersistentVolume(vmRef ref.Ref, persistentVolumes []cor
 	return
 }
 
-func GetOvaPvNfs(client client.Client, planName string, providerName string) (pv *core.PersistentVolume, found bool, err error) {
-	pv = &core.PersistentVolume{}
-	err = client.Get(
+func GetOvaPvNfs(dClient client.Client, planID string) (pvs *core.PersistentVolumeList, found bool, err error) {
```

Comment:
the name suggests that this function returns a single PV while we return a list - renaming the function makes more sense to me than returning the first result or something like that

Reply:
agreed, changed the name.